.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -98,6 +98,20 @@ The integration reached or maintains the following [Integration Quality Scale][q
 - [ ] 🥇 Gold
 - [ ] 🏆 Platinum
 
+<!--
+  This project is very active and we have a high turnover of pull requests.
+  Unfortunately the number of incoming pull requests is higher than what our
+  reviewers can review and merge so there is a long backlog of pull requests
+  waiting for review. You can help here! By reviewing another pull request
+  you will help raise the code quality of that pull request and the final
+  review will be faster. This way the general pace of pull request review
+  will go up and your wait time will go down. Thanks for helping out!
+-->
+
+To help with the load of incoming pull requests:
+
+- [ ] I have reviewed two other open pull requests in this repository.
```

Comment:
```suggestion
To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests](https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+sort%3Aupdated-desc+-author%3A%40me+) in this repository.
```

Reply:
Should we add the link here?

Reply:
I'll fix the sorting too.

Reply:
Now sorting on oldest created PR first.

Reply:
How about:

`is:open is:pr -author:@me -draft:true -label:waiting-for-upstream sort:created-asc -review:approved`

Filters out the drafts, the waiting ones and the already approved ones?


```suggestion
To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

prs: https://github.com/home-assistant/core/pulls?page=5&q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-asc+-review%3Aapproved
```

Reply:
Nice!

Reply:
Sorry about the broken link 😟